news/11527.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+This change restores 22.2.x wheel cache behavior to pip allowing the
+cache to find existing entries.
```

Comment:
```suggestion
Wheel cache behavior is restored to match previous versions, allowing the
cache to find existing entries.
```

This fragment appears in the changelog, and the sentence starting with _This change_ does not make sense to me.